subsys/net/lib/download_client/src/download_client.c

```diff
@@ -350,7 +350,7 @@ static int client_socket_connect(struct download_client *dl, int type, uint16_t
 			sin_addr = &((struct sockaddr_in *)&dl->remote_addr)->sin_addr;
 		}
 		inet_ntop(dl->remote_addr.sa_family, sin_addr, ip_addr_str, sizeof(ip_addr_str));
-		LOG_INF("Connecting to %s", ip_addr_str);
+		LOG_DBG("Connecting to %s", ip_addr_str);
```

Comment:
This information may be interesting to default have in a log. It shows that it's trying to connect to a host, and which IP address has been resolved. This is not a warning, I think we should keep as info.

Reply:
The host name has already been present, so I think this is way too verbose by default:
```
[00:02:54.629,364] <inf> fota_download_util: Download url coaps://eu.iot.avsystem.cloud:5674/fw/ZvekS3eD5wO8zA2oZynly5/mfw_nrf9160_update_from_1.3.6_FOTA_TEST_to_1.3.6.bin
[00:02:54.629,486] <inf> fota_download_util: Download Path fw/ZvekS3eD5wO8zA2oZynly5/mfw_nrf9160_update_from_1.3.6_FOTA_TEST_to_1.3.6.bin host coaps://eu.iot.avsystem.cloud:5674
[00:02:54.631,042] <inf> download_client: Downloading: fw/ZvekS3eD5wO8zA2oZynly5/mfw_nrf9160_update_from_1.3.6_FOTA_TEST_to_1.3.6.bin [0]
[00:02:54.756,500] <wrn> download_client: Failed to resolve hostname eu.iot.avsystem.cloud on IPv6
[00:02:54.757,385] <inf> download_client: Setting up TLS credentials, sec tag count 1
[00:02:54.757,537] <inf> download_client: Connecting to 35.204.45.127
[00:02:55.512,084] <inf> download_client: Downloaded 512/14248 bytes (3%)
```

6 lines of debug print just for doing two API calls `getaddrinfo()` and `connect()`

Reply:
I think the logs in fota_download_util should be trimmed.

Using the download_client directly I find it useful to have the hostname, resource path and the resolved IP address.
```
[00:01:52.543,121] <inf> download_client: Downloading: https://eu.iot.avsystem.cloud:5664/fw/g9b5v9zKXSzkwa4sUYNHOx/mfw_nrf91x1_update_from_2.0.1_to_2.0.1_FOTA_TEST.bin [0]
[00:01:53.074,432] <inf> download_client: Setting up TLS credentials, sec tag count 7
[00:01:53.074,523] <inf> download_client: Connecting to 35.204.45.127
[00:01:58.471,862] <inf> download_client: Downloaded 2048/14776 bytes (13%)
```